build.xml

```diff
@@ -284,4 +284,31 @@
         </exec>
     </target>
 
+    <target name="check-correct-yaml-indents" description="Run yaml standards checks">
+        <exec
+                executable="${path.yaml-standards}"
+                logoutput="true"
+                passthru="true"
+                checkreturn="true"
+        >
+            <arg value="./.github/" />
+            <arg value="./docker/" />
+            <arg value="./microservices/product-search/config/" />
```

Comment:
why is there so many paths? would it be possible and better to implement skipping paths?